Classes/Bookmark v2/BookmarkViewModel.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  BookmarkViewModel.swift
+//  Freetime
+//
+//  Created by Hesham Salman on 11/5/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+class BookmarkViewModel: NSObject, ListDiffable {
```

Comment:
nit: `final`

also you don't need to subclass `NSObject`

Reply:
Beyond limiting this class from growing out of control through extensions and subclasses, is there a perf. benefit to `final`?